Exercise_sequential_fastqc/fastqc_arrayjob.slurm.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+#####Use after training##############
+##SBATCH -p short
+#####################################
+
+#####Use during training###############
+#SBATCH --partition=reservation
+#SBATCH --reservation=fall2022_cpu_training
```

Comment:
reservation name should be fall_training_cpu_2022

Reply:
@ShobanaSekar , thanks for your comments and for catching that. It's been fixed now.